.github/CONTRIBUTING.md

```diff
@@ -1,13 +1,22 @@
 # Contributing
+![](https://raw.githubusercontent.com/checkstyle/resources/master/img/checkstyle-logos/checkstyle-logo-260x99.png)
 
-Please see instructions for contributing here:
-https://checkstyle.org/contributing.html
+#### [Reporting issues](https://checkstyle.org/report_issue.html)
+#### [Pull Request Template](https://github.com/checkstyle/checkstyle/blob/master/.github/PULL_REQUEST_TEMPLATE.md)
+#### [Issue Template](https://github.com/checkstyle/checkstyle/blob/master/.github/ISSUE_TEMPLATE.md)
+#### [How to report a bug?](https://checkstyle.org/report_issue.html#How_to_report_a_bug)
```

Comment:
`#` link is missing the final encoded character that takes user to the specific section. It should be `https://checkstyle.org/report_issue.html#How_to_report_a_bug.3F`.
Please confirm it is not just me who without that final character the link doesn't open to that specific section.

Reply:
@rnveach I am able to open the link without that final character. The link I am using is as below
https://checkstyle.org/report_issue.html#How_to_report_a_bug

Reply:
Its weird. It doesn't open to that specific section for me on firefox 63 and internet explorer without it (Windows and Lubuntu).

Reply:
I am using Chrome on an Ubuntu and it is working